osu.Game/Database/ModelManager.cs

```diff
@@ -200,6 +199,10 @@ public void Undelete(TModel item)
             });
         }
 
-        public abstract bool IsAvailableLocally(TModel model);
+        public virtual bool IsAvailableLocally(TModel model) => true;
+
+        public Action<Notification>? PostNotification { get; set; }
+
+        public virtual string HumanisedModelName => $"{typeof(TModel).Name.Replace(@"Info", "").ToLower()}";
```

Comment:
I think this may not technically be required, there's a default interface implementation that's the same:

https://github.com/ppy/osu/blob/6df4b16a892b7bc71c0cf40e48c7eb4ca58d0565/osu.Game/Database/IModelImporter.cs#L30

Looking away from `HumanisedModelName` as a whole in this pull, because as you say, it needs more work and is probably best addressed separately.